src/Microsoft.AspNetCore.SystemWebAdapters.CoreServices/RemoteAppClientOptions.cs

```diff
@@ -53,7 +53,14 @@ public Uri RemoteAppUrl
     }
 
     /// <summary>
-    /// Gets or sets an <see cref="HttpMessageHandler"/> to use for making requests to the remote app.
+    /// Gets or sets an <see cref="HttpMessageHandler"/> to use for making requests to the remote app. Used if BackchannelClient is null.
     /// </summary>
     public HttpMessageHandler? BackchannelHandler { get; set; }
+
+    /// <summary>
+    /// Gets or sets an <see cref="HttpClient"/> to use for making requests to the remote app.
+    /// </summary>
+    public HttpClient BackchannelClient { get; set; }
+        // Set to default, because we'll ensure this is populated with post-configuration
```

Comment:
But it won't be populated for users interacting with this object before that. Nullable is more accurate. Your consuming code already handles the null case.

Reply:
Good point. I'll just make this nullable.